src/main/java/com/rcv/Tabulator.java

```diff
@@ -752,27 +775,29 @@ private OvervoteDecision getOvervoteDecision(Set<String> candidateIDSet) {
           // If this fails, it means the code failed to handle an overvote with multiple
           // continuing candidates.
           assert selectedCandidate == null;
```

Comment:
Yeah, that's what I thought, but it looks like there's no error handling logic in here, so does it just crash the entire process?

Reply:
https://stackoverflow.com/questions/3018683/what-does-the-assert-keyword-do

assert needs to be enabled at launch by setting a flag  (-ea in the VM options in the run configuration) in which case yes, it would crash the application here.  I think the point with assertion is to catch programming logic errors and fix them, unlike like exceptions which would have error handling logic.  But it does seem like we should be running with the flag for it to be of any use.  I would have thought it was enabled by default.